app/models/alchemy/page/page_natures.rb

```diff
@@ -33,6 +33,15 @@ def contains_feed?
       definition["feed"]
     end
 
+    def scoped?
+      definition["editable_by"]
+    end
+
+    def scoped_user
```

Comment:
We should rename this method into something like:

`limited_editor_roles` or `editor_roles`

Something that reflects the purpose and adds a hint of the return value.